### ISSUE TITLE
Contributing Guidelines

### ISSUE THREAD

**Issue by reneklacan** [fetlife/fetlife-android#101]
I think it would be awesome to have them

https://github.com/blog/1184-contributing-guidelines

**Comment by Daddy** [comment]
Agreed. Especially as the app becomes larger and more widely used, we want guidelines set forth to keep contributions organized and efficient.

**Comment by JohnBaku** [comment]
@reneklacan true story... we use them for some of our other private repos... So having them for this repo sounds like a good idea!

We need to OS more stuff... I vote we open source vinyl! :-) #NotJoking

**Comment by kistitan** [comment]
hmm sounds good to me
